controllers/util/drpolicy_util.go

```diff
@@ -55,6 +55,17 @@ func DrpolicyRegionNamesAsASet(drpolicy *rmn.DRPolicy, drClusters []rmn.DRCluste
 	return sets.NewString(DrpolicyRegionNames(drpolicy, drClusters)...)
 }
 
+func IsMetro(drpolicy *rmn.DRPolicy, drClusters []rmn.DRCluster) bool {
+	regionNames := DrpolicyRegionNames(drpolicy, drClusters)
+	for _, regionName := range regionNames {
+		if regionName != regionNames[0] {
```

Comment:
This will not work for the case where regionNames[1] != regionNames[2]. There should only be on region in the whole list.